nulib/react-media-player
# Update build and packaging config

2 main victories of this PR are:
- More standard `tsconfig.json` file
- Package now builds and imports successfully

![image](https://user-images.githubusercontent.com/3020266/134393334-c4619690-ba2f-431c-8885-4e85f422f6d2.png)


To test a local import:

1. In this local project directory run `yarn link`
2. Fire up a test consuming app.  Or start a new Create React App if you want `npx create-react-app throwaway-consumer`
3. In consuming app terminal window, locally import our dev component.  `npm i /path/to/your/react-media-player` 
4. Copy and paste the output the `yarn link` command gives to link your local package.

If you're using create react app, you'll see an Invalid Hook error cause due to duplicate React copies being referenced.  We'll make our `react-media-player` code use the consuming app's `react` package.

`cd` back to the `react-media-player` directory and run:

```
npm link ../path/to/consuming/app/node_modules/react
```

Back in the consuming app import our component:
```
import Viewer from "react-media-player"
```

And use it somewhere.  Now a Create React App tester app should render the component.
